UserExperience_Remove_CssJs_Page_View.js

```diff
@@ -11,14 +11,19 @@ jQuery(function() {
 		'click',
 		'#w3tc_remove_cssjs_singles_add',
 		function() {
-			var single_path = prompt('Enter CSS/JS URL.');
+			var singlePath = prompt('Enter CSS/JS URL.');
```

Comment:
`const` is now recommended for this use instead of `var`.  See https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#declaring-variables-with-const-and-let

Reply:
All vars have been changed to const